styles.css

```diff
@@ -774,25 +944,564 @@ button:hover {
+  .about button {
+    margin-top: 20%;
+  }
+
+  .fa {
+    overflow-y: hidden;
+  }
+}
+
+@media only screen and (min-width: 768px) {
   html {
     overflow-x: hidden;
     width: 100%;
   }
 
+  body {
+    overflow-x: hidden;
+    width: 100%;
+    overflow-y: hidden;
+  }
+
+  nav {
+    visibility: hidden;
+  }
+
+  .nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    margin-top: 5%;
+  }
+
+  h1 {
+    margin-top: 10%;
+  }
+
+  .nav ul li {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 10px;
+    gap: 5px;
+    margin-top: 10%;
+  }
+
+  #break {
+    visibility: hidden;
+  }
+
+  .fa {
+    position: relative;
+    margin-top: 1000%;
+    overflow-y: hidden;
+  }
+
+  #above-work {
+    visibility: hidden;
+  }
+
+  .works {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-top: -1%;
+  }
+
+  .headline {
+    background-image: url(icons/Header-llustration-desktop.svg);
+    background-position: initial;
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0;
+    margin-top: -10%;
+    padding: 0;
+    height: 100%;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+    margin: 10%;
+  }
+
+  .headline p {
+    font-size: 14px;
+    font-weight: 300px;
+    line-height: 20px;
+    width: 70%;
+    align-self: center;
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    margin: 12%;
+    margin-top: 20%;
+  }
+
+  .containers {
+    width: 33.33%;
+  }
+
+  .container2 {
+    width: 33.33%;
+  }
+
+  .container3 {
+    width: 33.33%;
+  }
+
+  .contact1 {
+    position: absolute;
+    justify-content: center;
+    align-items: center;
+    visibility: visible;
+  }
+
+  .contact1 ul li {
+    width: 30%;
+    height: 10%;
+    border-radius: 0%;
+  }
+
+  .about-container img {
+    width: 60%;
+    background: none;
+  }
+
+  .contact1 ul li img {
+    width: 12px;
+  }
+
+  .next {
+    background-color: #fff;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  .h2 {
+    margin: 0;
+  }
+
+  .container1 {
+    width: 30.5%;
+    border: 1px solid #d0d9d4;
+    top: 7%;
+    margin: 5px;
+    padding: 0;
+    justify-content: center;
+    align-content: center;
+  }
+
+  .containera {
+    width: 30.5%;
+  }
+
+  .containerb {
+    width: 30.5%;
+  }
+
+  .containerc {
+    width: 30.5%;
+  }
+
+  .containerd {
+    width: 30.5%;
+  }
+
+  .containere {
+    width: 30.5%;
+  }
+
+  .text h2 {
+    width: 90%;
+    margin: 0;
+  }
+
+  .text ul {
+    width: 100%;
+    gap: 5px;
+    justify-content: center;
+  }
+
+  .text ul li {
+    width: fit-content;
+    padding: 5px;
+    padding-left: 0;
+    gap: 3px;
+  }
+
+  .about {
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement as I mentioned above regarding your design not matching properly with the Figma design. Your design should start responsive from the `768px` screen size. Right now because as I can see you are not using the Correct `layout`, `media query`, and `spacing` around the About section and the font size of the About me section content is not correct. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table. 

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/BHObserver/portfolio/assets/66476329/77165d6c-c76c-4e4c-8a54-e2400cce1380)

</td>
<td>

![Screenshot from 2023-06-17 18-51-12](https://github.com/tsigewossne/portfolio/assets/66476329/15e4af25-57ca-4a93-b595-03797bcb1f1e)

</td>
</tr>
</table>